### PR TITLE
Feature/deleting and favoriting ideas

### PR DESCRIPTION
**Pull request by stephaniemagdic**
#What does this PR do?
This PR added functionality to delete-button so that when the button is clicked the data model is updated and the new array is displayed on the page.
This PR also cleaned up commented out code that was no longer needed.

#Where should the reviewer start?
The reviewer should look to check the delete button functionality to check that everything is still working properly.

#Next Steps:
Start with the favorite button functionality and putting an event listener on the window to reassign savedIdeas array to localStorage.
Favorite cards with the updateIdeas method.

#Any background context you want to provide?
NA

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,11 +36,11 @@ <h2>Filter Starred Ideas</h2>
           <header>
               <button id="favorite" class="favorite-button">
                 <img name="star-white" src="assets/star.svg" alt="star">
-                <img name="star-red" src="assets/star-active.svg" alt="star">
+                <img name="star-red" id="starRed" src="assets/star-active.svg" alt="star">
               </button>
-              <button id="deleteCard" class="favorite-button">
+              <button id="deleteCard" class="delete-button">
```
Comment (joanafbrito) [inline index.html:41]: We should double check this button on the CSS file to make sure we did the change in every place where this information is being accessed `  <button id="deleteCard" class="delete-button">`

File: main.js [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
-
 var savedIdeas = [];
+
+// window.addEventListener("load", getLocalStorage);
+//
+// function getLocalStorage() {
+// for(var i =0; i < localStorage.length; i++){
+//   console.log(localStorage.getItem(localStorage.key(i)))
+// //   savedIdeas.push(JSON.parse(localStorage.getItem(localStorage.key(i))));
+// // savedIdeas.push(JSON.parse
```
Comment (joanafbrito) [inline main.js:22]: we should pay attention to line 22,  delete the unnecessary code, it is part duplicated.

File: main.js [hunk 3]
```diff
@@ -23,10 +34,13 @@ bodyInput.addEventListener("keyup", checkInputs);
...
-// xRedImg.addEventListener("click",);
 
+ideaContainerSection.addEventListener("click", function(e) {
+if(e.target.id === "x-red"){
+  deleteIdea(e)
+}
+}
+);
 
 function checkInputs() {
```
Comment (joanafbrito) [inline main.js:45]: We should review the name of this function, not sure about the plural.

File: main.js [hunk 4]
```diff
@@ -102,42 +110,18 @@ function displayIdeas() {
...
-
-
-
-
-
-
-
-
+  displayIdeas();
+  console.log("I have just called displayIDeas()");
```
Comment (joanafbrito) [inline main.js:125]: We should not forget to delete this console.log on line 125

**Review by joanafbrito — APPROVED** [review]
I have seen our code and we have a feel things to pay attention tomorrow, would be great start deleting unnecessary things and moving from that on.

Overall, great job today!

**Review by Logandv3 — APPROVED** [review]
I agree with Joana's comments.  Let's do it!

**Pull request merged**